bin/brew

```diff
@@ -13,6 +13,8 @@ fi
 BREW_FILE_DIRECTORY="$(chdir "${0%/*}" && pwd -P)"
 HOMEBREW_BREW_FILE="$BREW_FILE_DIRECTORY/${0##*/}"
 
+export HOMEBREW_PREFIX=$(cd $(dirname $(dirname $HOMEBREW_BREW_FILE)) && pwd -P)
```

Comment:
Thanks. Also note that you can avoid (weird) aliases by prepending a command with `\`; e.g.:
```shellSession
$ alias cd=echo
$ cd foo
foo
$ \cd foo
bash: cd: foo: No such file or directory
```